src/RunTests/README.md

```diff
@@ -0,0 +1,39 @@
+# Microsoft.Build.RunVSTest
+
+The `Microsoft.Build.RunVSTest` MSBuild SDK adds support for running tests from MSBuild, similarly to how one would use `dotnet test`.
+
+## Usage in `Directory.Packages.Props`
+In your global.json add the following:
+```json
+{
+  "msbuild-sdks": {
+	"Microsoft.Build.RunVSTest": "1.0.0"
```

Comment:
You shouldn't have it in both global.json and as a global package reference. You'd use it as one or the other.

In your case I don't see any Sdk.props/targets, so you can remove the global.json section.

Reply:
Updated to show usage for both Sdk and package reference